package.json

```diff
@@ -137,7 +137,10 @@
 		"translate:dependencies": "rm -rf build/dependency-strings && mkdir -p build/strings && wp-babel-makepot './node_modules/@wordpress/**/build-module/**/*.js' --ignore '**/*.native.js' --base './' --dir './build/dependency-strings' --output './build/strings/calypso-dependencies.pot'",
 		"translate:calypso": "wp-babel-makepot './{client,packages,apps}/**/*.{js,jsx,ts,tsx}' --ignore '**/node_modules/**,**/test/**,**/*.d.ts' --base './' --dir './build/strings' --output './public/calypso-strings.pot'",
 		"tsc": "NODE_OPTIONS='--max-old-space-size=4096' tsc",
-		"typecheck": "yarn run tsc --project client",
+		"typecheck": "run-p 'typecheck-*'",
+		"typecheck-client": "yarn run tsc --project client",
+		"typecheck-apps": "find apps -name tsconfig.json | xargs -n 1 -P 4 -I {} yarn tsc --noEmit --project {}",
+		"typecheck-packages": "tsc --build packages/tsconfig.json",
```

Comment:
should these be `typecheck:something` there's a bit of everything here so it's not like it's consistent but 🤷

Reply:
> should these be `typecheck:something` there's a bit of everything here so it's not like it's consistent but 🤷

I considered it, and would personally prefer `:` demarcations, but used `-` here because there's some prior art on folder commands specifically (`build-desktop`, `build-client`, `test-desktop`, `test-client`, etc.).

Our use of `:` seems to be more for "sub-commands", e.g. `:watch`/`:start` vs. `:build`, etc.